demo/universal/native/lib/MelRaw.re

```diff
@@ -1,13 +1,12 @@
-let%browser_only mockInitWebsocket = () => [%mel.raw
-  {|
+let%browser_only mockInitWebsocket = () =>
+  {%mel.raw |
   function mockInitWebsocket() {
     console.log("Load JS");
   }
-|}
-];
+|};
 
-let%browser_only initWebsocket = () => [%mel.raw
-  {|
+let%browser_only initWebsocket = () =>
+  {%mel.raw |
```

Comment:
Let's wait to merge this PR, until this is release: https://github.com/reasonml/reason/pull/2809

Reply:
Excellent, I found it weird as well.

Reply:
@davesnx Can we pin it until the release?

Reply:
Sure, there are no pins in this repo but go for it

Reply:
For anyone reading this, we decided to wait for the update.